### PR TITLE
README: update setup instructions.

### PR DESCRIPTION
**Pull request by mookid**
Use core.pager.
Add alternative setup via the command line.

see #67

cc @waldyrious

### CODE REVIEW

**Comment by mookid** [comment]
@waldyrious do you thing it looks good, or would you add something else while we are at it?

File: README.md [hunk 1]
```diff
@@ -66,20 +66,24 @@ and writes to stdout.
...
-    diff = path/to/diffr | less -R
+[core]
+    pager = diffr | less -R
 [interactive]
-    diffFilter = path/to/diffr
+    diffFilter = diffr
+```
+
+Alternatively, you can run from the command line:
+```
```
Comment (waldyrious) [inline README.md:85]: Just a small nit: other code blocks have a blank line above them, so this should probably have one too.

Comment (waldyrious) [reply README.md:85]: Other than that, the changes look good to me! 👍

Comment (mookid) [reply README.md:85]: thanks for the review.

**Pull request merged**